thanks.md

```diff
@@ -2,5 +2,17 @@
-- Edit this file, and tell us why you like Rubberduck. - [@username](https://github.com/username)
+<hr>
+
+I've been using VBA for as long as I can remember, but never knew anything other than a Making-It-Work coding style. I didn't give any thought to structuring my code. Needless to say I wasn't progressing <s>very fast</s> at all. I had a colleague introduce me to C# and tell me all that it could do that VBA couldn't. Everything he told me sounded great, I just had one little problem. I couldn't [grok](https://en.wikipedia.org/wiki/Grok) the concepts of Abstraction, Encapsulation, Inheritance, and Polymorphism. It nebulous thing I couldn't pin down. None of what I would read made any sense. I'd try to read and educate myself only to get bogged down in nomenclature, jargon and worst of all pedantry. Quickly I'd get lost down in the rabbit hole of terminology and have a not just a Fail-Fast moment but a Fail-Immediately moment.
+
+In pursuit of learning C# and to improve my coding I started posting on [CodeReview](https://codereview.stackexchange.com/). As luck would have it my first post got a comment from one of the co-founders [@Mat's Mug](https://github.com/retailcoder), as he's know at the [war pond](https://chat.stackexchange.com/rooms/14929/vba-rubberducking). He and the others at the pond helped me take what little I understood of Excel and VBA, or thought I understood, and use it to bridge the gap to understand C# concepts. After lots of reading, testing, trying, failing, failing, and even more failing things ever so slowy started to click and come together. I began to understand some of what I'd read about before.
+
+Using Rubberduck and contributing to it have been instrumental in learning. Nearly every day I learn something new or I'm able to connect the dots to better understand the greater programming picture. Code inspections to helped me to finally understand the differences between `ByVal` & `ByRef`. Learning about and successfully writing unit tests enables me to make changes without fear of unintentionally breaking functionality. The differences between Early and Late binding. Dependency Injection. Building, Cleaning, and Rebuilding a solution. Programming to an abstraction and not a concretion. Refactoring. Self documenting code (comments should explain the <i>why</i> not <i>what</i>). Consistent indentation. Referencing external assemblies. And everything else I've learned from Rubberduck...
+
+When I first started I didn't think I'd be able to successfully contribute, and honestly still don't. But if I didn't start when I did I wouldn't be where I'm at now. I'm glad my past self (exactly 1 year ago yesterday) was willing to step outside my comfort zone and ask about contributing to OSS. Just remember that you #GottaStartsSomewhere.
+
+Thanks to all those involved with RD enough for helping me learn a better way to code.
+
+[@IvenBach](https://github.com/IvenBach) 
```

Comment:
Should there be a date next to the name?

Reply:
Is theoretically visible in the blame, but might as well add it